CHANGELOG.md

```diff
@@ -26,6 +26,15 @@
 - Move `FragmentKey` into `android` package.
 - Move BaseFormulaFragment into `android` package.
 - Move `DisposableScope` into `android` package.
+- Move `FormulaFragment` into `android` package.
+- Move `BackCallback` into `android` package.
+- Move `FragmentLifecycleEvent` into `android.events` package.
+- Move `Integration` into `android` package.
+- Move `FragmentLifecycleCallback` into `android` package.
+- Move `FragmentComponent` into `android` package.
```

Comment:
I accidentally merged #205 into this PR as well. Sorry @Jawnnypoo

Reply:
Oops, no worries